t/configuration-loading-lazy.t

```diff
@@ -75,7 +76,7 @@ env THREESCALE_PORTAL_ENDPOINT=http://127.0.0.1:$TEST_NGINX_SERVER_PORT;
 GET /t?user_key=fake
 --- error_code: 200
 --- user_files eval
-[
+[ 
```

Comment:
:eyes:

Reply:
just empty whitespace on the end of the line. I think that it's good to have it clean :-)